realm/realm-library/src/objectServer/java/io/realm/SyncConfiguration.java

```diff
@@ -411,19 +477,27 @@ public String getUrlPrefix() {
     /**
      * Returns what happens in case of a Client Resync.
      */
-    public ClientResyncMode getClientResyncMode() {
+    ClientResyncMode getClientResyncMode() {
         return clientResyncMode;
     }
 
+    /**
+     * Returns the value this Realm is partitioned on. The partition key is a property defined in
+     * MongoDB Realm. All classes with a property with this value will be synchronized to the
+     * Realm.
+     *
+     * @return the value being used by MongoDB Realm to partition the server side MongoDB Database
+     * into smaller Realms that can be synchronized independently.
```

Comment:
maybe remove _smaller_ since it's subjective 
```suggestion
     * into Realms that can be synchronized independently.
```